### PR TITLE
Modified readme to reflect usage

### PR DESCRIPTION
**Pull request by bkbonner**
changed example to reflect the actual directive and added some text to point the user to the openweathermap.org info for an api

### CODE REVIEW

**Comment by bkbonner** [comment]
Actually, I delved into this a bit further, and I corrected the directive name based upon your code, but the usage doesn't take a city as you originally posted, it looks like it takes an account with lat/long.

Looks like the directive might still needs some cleanup..

File: README.MD [hunk 1]
```diff
@@ -18,7 +19,7 @@ angular.module('myApp', ['angular-weather']);
 
 - User the directive to display the current weather in somewhere
 ```html
-<angular-weather city="Houston"></angular-weather>
+<ng-weather-temperature city="Houston"></ng-weather-temperature>
```
Comment (ceoaliongroo) [inline README.MD:22]: Please remove, the name will keep as `angular-weather`.

**Comment by ceoaliongroo** [comment]
@bkbonner, thanks for take time and review the directive. Yes, you're write the module need some code, you will see soon.

Could do the change to merge your README comment about the API?

**Comment by ceoaliongroo** [comment]
@bkbonner the code is updated. You can download the versión and test it.

**Comment by ceoaliongroo** [comment]
Thanks

**Pull request merged**